zerver/openapi/zulip.yaml

```diff
@@ -11648,6 +11664,14 @@ paths:
                             description: |
                               Whether to [translate emoticons to emoji](/help/enable-emoticon-translations)
                               in messages the user sends.
+                          display_emoji_reaction_users:
+                            type: boolean
+                            description: |
+                              Whether to display the names of reacting users on a message.
+
+                              When this setting is turned on, it shows the names of reacting users
+                              on a message when the total reactions are 3 or less; and reverts back
+                              to showing the count when they are more than 3.
```

Comment:
Something is wrong with how these changes are appearing -- they seem all next to each other in a way that is surely wrong.